README.md

```diff
@@ -168,6 +169,20 @@ with the command `pca-wizard-templates  --emails`.
 | interface_type | Type of interface GoPhish will use with mail server. | string | "SMTP" | no |
 | ignore_cert | Indicate if GoPhish should ignore certs with mail server. | boolean | `True` | no |
 
+## User Report Field Dictionary ##
+
+User report JSONs are also exported by ```gophish-export```. User reports
+summarize data from targeted user clicks on phishing emails generated by a
+campaign. These JSON files are later imported into the PCA database.
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| customer | A corresponding customer ID that this user report document is associated with. It is left null as it will be filled in later by a PCA operator or another tool prior to importation | string | null | No |
+| assessment | A corresponding assessment ID that this user report is associated with. | string | | yes |
+| campaign | A corresponding campaign ID that this user report is associated with. | string| | yes |
+| first_report | The first report (click) generated by a targeted user. Formatted by YYYY-MM-DDT: hh:mm.ss"| datetime | | yes |
+| total_num_reports | The total number of user reports received during a campaign.| integer | | no
```

Comment:
I put them in this order because this is the order I have them on in the JSON file. Are you sure you wanna change the field order?

Reply:
The order of the fields in the JSON file shouldn't be relevant, right?  It isn't a CSV.  I think @dav3r is right here.

If part of your code depends on the order of JSON dictionary keys that probably indicates a problem, since JSON does not make any promises about such ordering.  If you parse a JSON and then re-export it the order of the keys can change depending on the hash map implementation being used internally.

Reply:
See bac2cb88e00b183bdbae8f990c3ad5a48b01ef05

Reply:
@JCantu248, https://github.com/cisagov/gophish-tools/pull/33/commits/bac2cb88e00b183bdbae8f990c3ad5a48b01ef05 does not correct the alphabetical order.

Also, please use more descriptive messages in your commits: `Merging remote` is not particularly helpful here.  We strongly encourage everyone to follow the commit message style and guidelines described in [this post](https://chris.beams.io/posts/git-commit/#seven-rules), as referenced in our [Development Guide](https://github.com/cisagov/development-guide#git-github-and-you).

Reply:
Thanks @JCantu248 for fixing the alphabetization in https://github.com/cisagov/gophish-tools/pull/33/commits/cc24b7780b4ff24ca8f43633d2dce6a214d86892. 👍